<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>C1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>157</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Re</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QLineEdit" name="Num1_Real_lineEdit"/>
    </item>
    <item row="0" column="9">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Im</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11" colspan="2">
     <widget class="QLineEdit" name="Num1_Ima_lineEdit"/>
    </item>
    <item row="0" column="13">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>C2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>151</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Re</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" colspan="2">
     <widget class="QLineEdit" name="Num2_Real_lineEdit"/>
    </item>
    <item row="1" column="9">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="10">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Im</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11" colspan="2">
     <widget class="QLineEdit" name="Num2_Ima_lineEdit"/>
    </item>
    <item row="1" column="13">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" rowspan="2">
     <spacer name="horizontalSpacer_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Resultado</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="2">
     <spacer name="horizontalSpacer_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>90</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Re</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" rowspan="2" colspan="2">
     <widget class="QLineEdit" name="Res_Real_lineEdit"/>
    </item>
    <item row="2" column="9" rowspan="2">
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="10">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Im</string>
      </property>
     </widget>
    </item>
    <item row="2" column="11" rowspan="2" colspan="2">
     <widget class="QLineEdit" name="Res_Ima_lineEdit"/>
    </item>
    <item row="2" column="13" rowspan="2">
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" rowspan="2" colspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>97</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="8">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>90</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="12">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>90</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QPushButton" name="Suma_pushButton">
      <property name="text">
       <string>Suma</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Resta_pushButton">
      <property name="text">
       <string>Resta</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="3">
     <widget class="QPushButton" name="Mul_pushButton">
      <property name="text">
       <string>Mul</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QPushButton" name="Conjugado_pushButton">
      <property name="text">
       <string>Conjugado</string>
      </property>
     </widget>
    </item>
    <item row="5" column="9" colspan="3">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>87</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="12">
     <widget class="QPushButton" name="Close_pushButton">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="5" column="13">
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
